specification/redis/resource-manager/Microsoft.Cache/stable/2020-12-01/redis.json

```diff
@@ -2397,7 +2397,8 @@
         "maintenanceWindow": {
           "type": "string",
           "format": "duration",
-          "description": "ISO8601 timespan specifying how much time cache patching can take. "
+          "description": "ISO8601 timespan specifying how much time cache patching can take.",
+          "default": "PT5H"
```

Comment:
is it kind of read-only property, as user is not setting it?

Reply:
Actually the user can set this property, it's the existence of its default value that raised the issue

Reply:
thanks for info.